package.json

```diff
@@ -62,8 +62,8 @@
     "@deephaven/eslint-config": "^0.41.0",
     "@deephaven/prettier-config": "^0.41.0",
     "@jupyterlab/builder": "^3.6.5",
```

Comment:
I couldn't get the same error as you, but I did get an error about needing to increase this version:
```sh
$ jupyter labextension build --development True .
Building extension in .
/home/bender/.local/lib/python3.10/site-packages/jupyterlab/debuglog.py:56: UserWarning: An error occurred.
  warnings.warn("An error occurred.")
/home/bender/.local/lib/python3.10/site-packages/jupyterlab/debuglog.py:57: UserWarning: ValueError: Extensions require a devDependency on @jupyterlab/builder@^4.0.2, you have a dependency on 3.6.6
  warnings.warn(msg[-1].strip())
/home/bender/.local/lib/python3.10/site-packages/jupyterlab/debuglog.py:58: UserWarning: See the log file for details: /tmp/jupyterlab-debug-ji273yqd.log
  warnings.warn(f"See the log file for details: {log_path!s}")
error Command failed with exit code 1.
```

I changed the version of `@jupyterlab/builder` and was able to build then: `    "@jupyterlab/builder": "^4.0.2",`

Reply:
Though I'm not sure how that makes sense, since the ts-cookiecutter project is still at 3.0.0: https://github.com/jupyter-widgets/widget-ts-cookiecutter/blob/94f9d384331150655c18a08d94307280c32adedd/%7B%7Bcookiecutter.github_project_name%7D%7D/package.json#L58